docs/source/DeveloperGuide/ReleaseProcedure/index.rst

```diff
@@ -26,237 +29,42 @@ The release manager needs to:
-  $ git fetch release  # this will bring in the updated release/rel-v7r1 branch from the github repository
-  $ git rebase --no-ff release/rel-v7r1  # this will rebase the current rel-v7r1 branch to the content of release/rel-v7r1
-
-You can now proceed with pushing, and check the tests::
-
-  $ git push release rel-v7r1  # we push to the rel-v7r1 branch too.
-
-From the previous command, note that due to the fact that we are pushing a branch named *rel-v7r1*
-to the *release* repository, where it already exists a branch named *rel-v7r1*,
-**the local branch will override the remote one**.
-
-Now, before performing any further step, you should go to `GitHub Actions (GA) <https://github.com/DIRACGrid/DIRAC/actions>`_
-and check the result of the workflows that are running on the pushed *rel-v7r1* branch.
-
-If everything is fine, you can tag::
-
-  $ git tag -a v7r1p37 -m "v7r1p37"  # this will create an annotated tag, from the current branch, in the local repository
-  $ git push release v7r1p37  # we push to the *release* repository (so to GitHub-hosted one) the tag just created
-
-All the patches must now be also propagated to the *upper* branches.
-In this example we are going through, we are supposing that it exists *rel-v7r2* branch,
-from which we already derived production tags. We then have to propagate the changes done to
-*rel-v7r1* to *rel-v7r2*. Note that if even the patch was made to an upstream release branch, the subsequent
-release branch must also receive a new patch release tag. Multiple patches can be
-add in one release operation.::
-
-  $ git checkout -b rel-v7r2 release/rel-v7r2
-  $ git merge release/rel-v7r1
-
-This may result in merge conflicts, which should be resolved "by hand".
-One typical conflict is about the content of the ``release.notes`` file.
-
-From now on, the process will look very similar to what we have already done for
-creating tag v7r1p37. We should then repeat the process for v7r2::
-
-  $ vim release.notes
-  $ vim __init__.py
-
-Merge PRs (if any), then save the files above. Then::
-
-  $ git commit -a  # this will commit the changes we made to the release notes in rel-v7r2 local branch
-  $ git fetch release  # this will bring in the updated release/rel-v7r2 branch from the github repository
-  $ git rebase --no-ff release/rel-v7r2  # this will rebase the current rel-v7r2 branch to the content of release/rel-v7r2
-  $ git push release rel-v7r2  # we push to the *release* remote the tag just created, and the rel-v7r2 branch.
-
-Now, check GA and if everything is fine::
-
-  $ git tag v7r2p8  # this will create a tag, from the current branch, in the local repository
-  $ git push v7r2p8  # we push to the *release* remote the tag just created
-
-The *master* branch of DIRAC always contains the latest stable release.
-If this corresponds to rel-v7r2, we should make sure that this is updated::
-
-  $ git push release rel-v7r2:master
-
-Repeat the process for every "upper" release branch.
```

Comment:
You have removed all this part about the merging and propagation. Because of the sweeper getting in?

Reply:
I hope so, I think mixing merging docs with releasing docs is bad idea as they should be two separate concerns.

The sweeping should definitely be documented but that's a task for another PR IMHO.